env.js

```diff
@@ -0,0 +1,16 @@
+var clusters = [
+   {
+       NAME:"prod",
+       // Schema Registry service URL (i.e. http://localhost:8081)
+       SCHEMA_REGISTRY: "http://localhost:8081", // https://schema-registry.demo.landoop.com
+       // Avro4S backend url. Allows converting Avro -> Scala Case classes
+       AVRO4S : "https://platform.landoop.com/avro4s/avro4s",
```

Comment:
- AVRO4S Url does not work, though it may be needed for future features (as converting Avro -> Scala Case   classes). Moved to factory.
- 2-7 fixed
- 8 Just made the tree (a lot) wider, I dont thing its really needed to hide documentation column
- 1 I am working on that, not sure why it is happening. It has strange behaviour here as well: /#/cluster/prod/schema/yahoo-stocks-value/version/2